### PR TITLE
Travis: fix .deb installation

### PR DESCRIPTION
**Pull request by plata**
The stage works but the output shows:
```
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?
```

### CODE REVIEW

**Comment by plata** [comment]
@ImperatorS79 some of the dependencies do not seem to exist in Ubuntu 18.04. Any ideas?

**Comment by ImperatorS79** [comment]
All was working fine before. Something went wrong on the ubuntu side, because packages.ubuntu.com seems to be broken (no wine package :/)

**Comment by plata** [comment]
So just try again tomorrow or so?

**Comment by ImperatorS79** [comment]
Yes I think so

**Comment by plata** [comment]
I don't get how it should be done...

**Comment by ImperatorS79** [comment]
Everything works locally.

**Comment by plata** [comment]
I don't doubt that. I'm just wondering how to test it in Jenkins.

**Comment by plata** [comment]
Ok, got it... Didn't read the error message properly.

File: .travis.yml [hunk 1]
```diff
@@ -79,9 +79,25 @@ jobs:
...
+      services:
+        - docker
+      before_install:
+        - docker run -d --name ubuntu-18-04 openjdk:8 tail -f /dev/null
+      install:
+        - docker exec -t ubuntu-18-04 bash -c "dpkg --add-architecture i386;
+                                               apt -qq update;
+                                               apt install -y git;
+                                               apt install -y maven;
+                                               apt install -y openjfx;
```
[the remark this answers is not in the document]
Comment (plata) [reply .travis.yml]: copy + paste ...

**Pull request merged**